source/MRMesh/MRToolPath.cpp

```diff
@@ -191,12 +205,28 @@ std::vector<PlaneSections> extractAllSections( const Mesh& mesh, const MeshPart&
     return sections;
 }
 
-std::vector<IsoLines> extractAllIsolines( const Mesh& mesh, VertId startPoint, float sectionStep, ProgressCallback cb )
+std::vector<IsoLines> extractAllIsolines( const Mesh& mesh, VertId startPoint, const SurfacePath& startSurfacePath, float sectionStep, ProgressCallback cb )
```

Comment:
why `const SurfacePath& startSurfacePath` and not `const EdgePath& startEdgePath`